k2/csrc/tensor_ops.h

```diff
@@ -106,6 +109,26 @@ Tensor IndexAdd(Tensor &src, Array1<int32_t> &indexes, int32_t dim,
 void IndexAdd(Tensor &src, Array1<int32_t> &indexes, bool allow_minus_one,
               Tensor *dest);
 
+/*
+  Returns a 1-D Tensor that is a result of indexing 1-D `src` with Ragged array
+  `indexes` whose NumAxes() is 2. ans.Dims()[0] will equal to indexes.Dims0() as
```

Comment:
you mean Dim0()..

Reply:
Here ans is a `k2::Tensor` so I use `ans.Dims()`

Reply:
Oh, yes, sorry, will update